src/main/java/seedu/address/logic/parser/DeleteCommandParser.java

```diff
@@ -19,8 +21,14 @@
      */
     public DeleteCommand parse(String args) throws ParseException {
         try {
-            Index index = ParserUtil.parseIndex(args);
-            return new DeleteCommand(index);
+            ArrayList<Index> indexArrayList = new ArrayList<Index>();
+            String[] indexArray = args.split(" ");
+            // start looping from i = 1 as the first element in the array is an empty string
+            for (int i = 1; i < indexArray.length; i++) {
+                Index index = ParserUtil.parseIndex(indexArray[i]);
+                indexArrayList.add(index);
+            }
+            return new DeleteCommand(indexArrayList);
```

Comment:
I created a common method you could use once my PR gets merged:
https://github.com/CS2103AUG2017-W10-B2/main/blob/1b592d12a28c72b3288570f44de4484e2b9f172a/src/main/java/seedu/address/logic/parser/ParserUtil.java#L58
You only have to pass in `args` like so:
`return new DeleteCommand(ParserUtil.parseMultipleIndexes(args));`

---

@CS2103AUG2017-W10-B2/developers For anybody looking to implement new commands that requires parsing multiple indexes. :-)